canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -137,7 +138,7 @@ $palette-search-container-height: 41px;
 			width: 100%;
 			align-items: center;
 			height: 100%;
-			margin-left: 20px;
+			margin-left: $spacing-05;
```

Comment:
This is causing the palette icons to be further left than before - please revert back to 20px.